app/models/authorization.rb

```diff
@@ -0,0 +1,28 @@
+# encoding: utf-8
+
+class Authorization
+  include HalApi::RepresentedModel
+
+  attr_accessor :token
+
+  def initialize(token)
+    @token = token
+  end
+
+  def user_id
+    token.user_id
+  end
+
+  def to_model
+    self
+  end
+
+  def persisted?
+    false
+  end
+
+  def cache_key
```

Comment:
Does this need a cache key?  I thought `/authorization` resources weren't cached.